src/item.h

```diff
@@ -354,7 +356,7 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
     std::string info( bool showtext, std::vector<iteminfo> &dump ) const;
 
     /** Burns the item. Returns true if the item was destroyed. */
-    bool burn( fire_data &bd );
+    bool burn( fire_data &bd, bool can_spread = false );
```

Comment:
Made can_spread optional because burn() is called for when the player enters a fire and burns its clothes. 
Should I make a burn_clothes() that calls burn() with a negative bool?